imaginairy/schema.py

```diff
@@ -159,26 +182,26 @@ def __repr__(self):
 
 
 class ImaginePrompt(BaseModel):
-    prompt: Optional[List[WeightedPrompt]]
-    negative_prompt: Optional[List[WeightedPrompt]]
+    prompt: Optional[List[WeightedPrompt]] = None
+    negative_prompt: Optional[List[WeightedPrompt]] = None
```

Comment:
I tried to use the auto upgrade tool they had. Didn't do much. It added all these " = None " here.

Reply:
I didn't try running without them